src/NUnitFramework/framework/Constraints/Constraint.cs

```diff
@@ -304,7 +304,7 @@ public DelayedConstraint.WithRawDelayInterval After(int delay)
         /// and polling interval.
         /// </summary>
         /// <param name="delayInMilliseconds">The delay in milliseconds.</param>
-        /// <param name="pollingInterval">The interval at which to test the constraint.</param>
+        /// <param name="pollingInterval">The interval at which to test the constraint, in milliseconds.</param>
```

Comment:
It would be more consistent to rename the parameter into `pollingIntervalInMilliseconds` to match the delay one.

Reply:
If that’s an acceptable break, I’ll make that change next time I’m in front of a keyboard

Reply:
Previously I would have said yes, but thinking off it, if someone did `.After(100, pollingInterval: 10)` it would be a breaking change. So therefore all we can change is the comments.

Reply:
If it’s valuable enough, this method can be obsoleted in favor of the enhanced syntax `.After(100).Milliseconds.PollEvery(10).Milliseconds`. It would be the perfect scenario for an analyzer and fixer.